src/caffeine/caffeine.h

```diff
@@ -16,48 +16,48 @@ enum {
 
 // Program launch and finalization
 
-void caf_c_caffeinate(int argc, char *argv[]);
-void caf_c_decaffeinate(int exit_code);
+void caf_caffeinate(int argc, char *argv[]);
+void caf_decaffeinate(int exit_code);
```

Comment:
`prif_stop` calls `caf_decaffeinate`. Should the caffeine side names be updated as well to `caf_init` and `caf_stop` to represent more closely the relationship with `prif_init` and `prif_stop`?

Reply:
I'm not too worried about the names of "implementation" procedures. I would envision an implementation doing, for the sake of reducing procedure call overhead, something like

```
module prif
  use caffeine, only: &
    prif_init => caffeinate, &
    prif_stop => decaffeinate, &
    ...
end module
```

At least to the extent that the interfaces match up.